lcx-api/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "lcx-api",
+  "version": "0.1.0",
+  "license": "MIT",
+  "main": "index.js",
+  "scripts": {
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "test": "yarn mocha --timeout 0"
+  },
+  "dependencies": {
+    "@chainlink/ea-bootstrap": "0.0.1",
+    "@chainlink/external-adapter": "^0.2.3",
+    "mocha": "^8.1.3"
```

Comment:
Mocha dependency should not be needed here, we have it in the root `package.json`.

```suggestion
```